packages/base64/package.json

```diff
@@ -28,7 +28,8 @@
     "lint-fix": "eslint --fix '**/*.js'",
     "prepublish": "yarn clean && yarn build",
     "test": "ava",
-    "test:nyc": "nyc ava"
+    "test:nyc": "nyc ava",
+    "test:xs": "exit 0"
```

Comment:
seems odd that this one is different from the rest. Others seem to have `lint:*` stuff after `test`, but this one is alphabetical or something.